### PR TITLE
added install directions to the readme

### PR DESCRIPTION
**Pull request by tomlazar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,18 @@ Unofficial Ruby API to get data from The Blue Alliance.
...
 This API returns JSON-parsed data fetched from The Blue Alliance's API.
 
+## Installation
+
+Add `tba.rb` to your Gemfile.
+```ruby
+gem 'tba'
+```
+
+Or manualy install
```
Comment (ErikBoesen) [inline README.md:16]: *manually

Comment (ErikBoesen) [inline README.md:16]: "Or" is incorrect. You MUST do both of these.

**Review by ErikBoesen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tomlazar) [reply README.md:16]: if you add it to the Gemfile and run `bundle install` shouldn't that install it for you?

Comment (ErikBoesen) [reply README.md:16]: Yes, but it doesn't explain to do that.

**Pull request closed without merging**